rust/src/verify/mod.rs

```diff
@@ -267,12 +268,12 @@ impl<'a> Verifier<'a> {
         let parsed_tl_signature = parse_tl_signature(tl_signature)?;
 
         match &parsed_tl_signature.header.tl_version {
-            Some(tl_version) if tl_version == "1" => VerifierV1 {
+            None | Some(TlVersion::V1) => VerifierV1 {
```

Comment:
Why do we want `None` to be parsed as V1?

Reply:
If no version was provided, we assumed that it would be v1. 
https://github.com/TrueLayer/truelayer-signing/blob/b752b13309cfa27bb8bf367b46a3189bd141f0f0/rust/src/verify/mod.rs#L275